application/controllers/admin/Tokens.php

```
@@ -1062,68 +1062,48 @@ public function addDummies(int $iSurveyId, string $subaction = '')
+                $newDummyToken = 0;
+                while ($newDummyToken < $aData['amount'] && $invalidtokencount < 50) {
+                    $token = Token::create($iSurveyId);
+                    $token->setAttributes($aData, false);
+
+                    $token->firstname = str_replace('{TOKEN_COUNTER}', $newDummyToken, (string) $token->firstname);
+                    $token->lastname = str_replace('{TOKEN_COUNTER}', $newDummyToken, (string) $token->lastname);
+                    $token->email = str_replace('{TOKEN_COUNTER}', $newDummyToken, (string) $token->email);
+
+                    $token->generateToken($aData['tokenlength']);
+
+                    $existingtokens[$token->token] = true;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$existingtokens'. (undefined)

(UnusedLocalVariable)